qemu.go

```diff
@@ -862,11 +947,27 @@ func (q *qemu) hotplugDevice(devInfo interface{}, devType deviceType, op operati
 }
 
 func (q *qemu) hotplugAddDevice(devInfo interface{}, devType deviceType) error {
-	return q.hotplugDevice(devInfo, devType, addDevice)
+	if err := q.hotplugDevice(devInfo, devType, addDevice); err != nil {
+		return err
+	}
+
+	if err := q.pod.storage.storeHypervisorState(q.pod.id, q.state); err != nil {
```

Comment:
This can be improved later (moving this to container.go where we actually add the devices), but that's fine for now.